recipes/newrelic/infrastructure/kubernetes.yml

```diff
@@ -481,6 +481,13 @@ install:
               ARGS="${ARGS} --devel"
             fi
             ARGS="${ARGS} --set ksm.enabled=${NR_CLI_KSM}"
+
+            if [[ $SERVER_MAJOR_VERSION -eq 1 && $SERVER_MINOR_VERSION -lt 23 ]]; then
```

Comment:
this makes everything k8s >=1.24 use KSM 2.7.0?

this chart seems to imply that KSM 2.7.0 should be used for k8s >=1.25: https://github.com/kubernetes/kube-state-metrics#compatibility-matrix

Reply:
Hi @nrepai, great find! This actually makes everything k8s v>= 1.23 use KSM 2.7.0. Even though I tested k8s v1.23 with KSM v2.7.0, but it makes sense to follow the guidance from the compatibility matrix. I will make the change.